agent/gogenerate/inflections.csv

```diff
@@ -1 +1,2 @@
 Docker:
```

Comment:
Looks like `Docker:` is in the upstream [inflections.csv](https://github.com/aws/aws-sdk-go/blob/e039cdef4b5a23d86636b36936529cdd33cf122c/internal/model/api/inflections.csv#L1535), so you can likely remove it here.